cmd/snap/cmd_model_test.go

```diff
@@ -457,7 +457,7 @@ serial:          serialserial
 architecture:    amd64
 base:            core20
 timestamp:       2018-09-11T22:00:00Z
-snaps:
+snaps:                
```

Comment:
I'm not entirely sure what is happening here. Anyway the goal should be not to change these tests at all.

Reply:
probably the WordWrap thingy introduced this

Reply:
It was the common modelWriter I had written, it generalized the output instead of having handcrafted output which resulted in this extra '\t' added after 'snaps:', which in turn results in this large space because of tabWriter  that aligns text

Reply:
This PR will be closed as I will open new